lib/process-services/src/lib/process-list/components/start-process/start-process.component.scss

```diff
@@ -63,13 +78,28 @@
 
     &-start-form-actions:has(.adf-start-form-actions-button) {
         text-align: right;
+        padding-bottom: 16px;
+
+        button {
+            text-wrap: none;
+        }
     }
 
     &-start-process-definition-container {
         display: flex;
         flex-direction: row;
         justify-content: space-around;
         align-items: baseline;
+
+        .adf-start-process-definition-select {
+            .mat-mdc-select-arrow-wrapper {
```

Comment:
Can we add this selector in the mat-selectors scss file and then use the variable instead?

Reply:
Oh, I didn't see that the PR was merged already. I guess we can handle that cleanup at a later time then.